app/src/main/java/com/asfoundation/wallet/manage_wallets/ManageWalletFragment.kt

```diff
@@ -115,21 +116,30 @@ class ManageWalletFragment : BasePageViewFragment() {
       containerColor = styleguide_blue,
     ) { padding ->
       when (val uiState = viewModel.uiState.collectAsState().value) {
```

Comment:
this may not be collecting the state change of the uiState when a new wallet is created